roc.yml

```diff
@@ -0,0 +1,69 @@
+--- 
+GENERATOR: 
+  PackageName: roc 
+  PackageDescription: "Package roc provides Go bindings for libroc"
+  PackageLicense: ""
+  Includes:
+    - ../include/roc/address.h
+    - ../include/roc/config.h
+    - ../include/roc/context.h
+    - ../include/roc/frame.h
+    - ../include/roc/log.h
+    - ../include/roc/platform.h
+    - ../include/roc/receiver.h
+    - ../include/roc/sender.h
+  FlagGroups:
+    - {name: "CFLAGS", flags: [
+        "-I\"../include\"",
+    ]}
+    - {name: "LDFLAGS", flags: [
+      "-lroc",
+    ]} 
+    
+PARSER:
+  IncludePaths: ["include", "/usr/lib/gcc/x86_64-redhat-linux/9/include"]
+  SourcesPaths:
+    - include/roc/address.h
+    - include/roc/config.h
+    - include/roc/context.h
+    - include/roc/frame.h
+    - include/roc/log.h
+    - include/roc/platform.h
+    - include/roc/receiver.h
+    - include/roc/sender.h
```

Comment:
As well as this one?

Reply:
see my comment above. roc.yml will probably remain in the repo giving the possibility to generate the code. Why not leave it as is and quickly work on the next pr that includes pkg-config?